### ISSUE TITLE
How do we use "GetUser"?

### ISSUE THREAD

**Issue by bitmammoth** [blubwage/DarkRift2_Boilerplate#12]
I am having trouble pulling data from the database using this method. Everything works but I cannot figure out how to return the data from that specific interface.

**Comment by mwage** [comment]
Have you taken a look at one of the two database connector plugins? Should have examples for mysql and mongodb. Also, I'm surprised things still work, been years since I touched this repo. :P

**Comment by bitmammoth** [comment]
To be honest I was not expecting a reply so quickly or even ever :)

I have searched through all files but have not found a way to get that the rest of the account data with that interface I must be doing something wrong lol. 

About the repo, It took a little bit to get set up but it seems everything is still working without a flaw. I was very happy to find this code its one of the only remaining functional plugins for MySql and the only one I have seen for MongoDB. 

I really appreciate all that work you did years ago! hehe, I will keep trying to sort this out it's the last bit stopping my progress.

It's working on Unity 2020.3 and the Newest DR2 btw :)

EDIT: Got it I found the sample I needed by doing a search in the login plugin. Thank you again for your time!
